recipes/server.rb

```diff
@@ -272,24 +323,30 @@ def remove_job(job, servernode)
 jobs.each do |job|
 
   # Shorten some long variables for readability.
+  type = job['type']
   fqdn = job['fqdn']
-  sd = job['source-dir']
-  dd = File.join(job['destination-dir'], 'filesystem', fqdn, sd)
   suser = servernode['rdiff-backup']['server']['user']
```

Comment:
`server_user` would be more descriptive. `suser` is really on descriptive if you look at this line, but if you look at this line you already know what it is.